pkg/render/renderer_impl.go

```diff
@@ -323,13 +324,23 @@ func templateFuncs() htmltemplate.FuncMap {
 		"disabledIf":       disabledIf,
 		"t":                translate,
 		"passwordSentinel": pwdSentinel,
+		"hasOne":           hasOne,
+		"hasMany":          hasMany,
 
 		"rbac": func() map[string]rbac.Permission {
 			return rbac.NamePermissionMap
 		},
 	}
 }
 
+func hasOne(a []*interface{}) bool {
```

Comment:
Neither of these function signatures are generic enough unfortunately. You'll need to accept `interface{}` (`[]interface{}` is NOT the same as `[]T`, unfortunately), then use reflection to determine if it's an array. I already do this in another template helper, so you should be able to copy that or pull it into a helper.